web/themes/custom/pineapple/components/_card.twig

```diff
@@ -2,13 +2,29 @@
 {% set attributes = attributes|default(create_attribute()).addClass([
   'card-container',
 ]) %}
-<div {{ attributes }}>
+{% if card_type == 'vertical'%}
+  <div {{ attributes.addClass('vertical') }}>
+{% else %}
+  <div {{ attributes.addClass('horizontal') }}>
+{% endif %}
```

Comment:
You don't need to do this, if you add `card_type` into the classes array below card-container It'll achieve the same result without the if statements.

Reply:
done